docs/user/build/faq.rst

```diff
@@ -82,15 +84,18 @@ How do I create an executable jar for my GeoTools app?
 
 If you're familiar with Maven you might have used the `assembly plugin
 <http://maven.apache.org/plugins/maven-assembly-plugin/>`_ to create self-contained, executable jars. The bad news is
-that this generally won't work with GeoTools. The problem is that GeoTools modules often define one or more files in its
-META-INF/services directory with the same names as files defined in other modules.  The assembly plugin just copies
-files with the same name over the top of each other rather than merging their contents.
-
-The good news is that the `Maven shade plugin <http://maven.apache.org/plugins/maven-shade-plugin/index.html>`_ can be
-used instead and it will correctly merge the META-INF/services files from each of the GeoTools modules used by your
-application.
-
-The POM below will create an executable jar for the GeoTools :doc:`/tutorial/quickstart/index` module which includes all of the required
+that this generally won't work with GeoTools. The problem is that GeoTools
+modules often define one or more files in its ``META-INF/services`` directory
+with the same names as files defined in other modules.  The assembly plugin just
+copies files with the same name over the top of each other rather than merging
+their contents.
+
+The good news is that the `Maven shade plugin
+<http://maven.apache.org/plugins/maven-shade-plugin/index.html>`_ can be used
+instead and it will correctly merge the ``META-INF/services`` files from each of
+the GeoTools modules used by your application.
+
+The pom below will create an executable jar for the GeoTools :doc:`/tutorial/quickstart/index` module which includes all of the required
```

Comment:
What about:
The **POM** below will create an executable **JAR** for the GeoTools
?